src/Microsoft.ML.Legacy/Runtime/EntryPoints/MacroUtils.cs

```diff
@@ -7,6 +7,9 @@
 using System.Linq;
 using Microsoft.ML.Runtime.Data;
 
+// The warning #612 is disabled because the following code uses a lot of things in Legacy.Models while Legacy.Model is marked as obsolete.
```

Comment:
When are we going to rewrite the below code? Is there an issue tracking it?

Reply:
This one: https://github.com/dotnet/machinelearning/issues/1565.